test/images/agnhost/inclusterclient/main.go

```diff
@@ -72,6 +72,7 @@ func main(cmd *cobra.Command, args []string) {
 
 	c := kubernetes.NewForConfigOrDie(cfg).RESTClient()
 
+	//lint:ignore SA1015 false positive, `time.Tick` is used in a main function
```

Comment:
It is not a false positive here, SA1015 says 

```
Using time.Tick in a way that will leak. Consider using time.NewTicker, and only use time.Tick in tests, commands and endless functions
```

So we are using it in the main function which is not complying with the rule, so I'm not sure what would the alternative, but at least the reason given here is wrong.

Reply:
You can call is a noisy positive instead, but OP is right in that staticcheck isn't helpful by flagging this use of time.Tick.

> So we are using it in the main function which is not complying with the rule

Actually you have that backwards. Using it in a main function is one of the accepted places; staticcheck 2019.2 is simply not smart enough to figure out that this package is really a main function in disguise. It looks for `package main` and isn't aware of Cobra and assembling commands from multiple packages. The next release will fix that, c.f. https://github.com/dominikh/go-tools/commit/00664db7fdb567f0b2efbbcdb7d9ed23f7135468

Reply:
@mkumatag do you want me to reword the comment, or is it ok as-is?

Reply:
> @mkumatag do you want me to reword the comment, or is it ok as-is?

lets do that if you can.

Reply:
Done!